core/src/oned/ODCode93Patterns.cpp

```diff
@@ -0,0 +1,62 @@
+/*
+* Copyright 2016 Nu-book Inc.
+* Copyright 2016 ZXing authors
+*/
+// SPDX-License-Identifier: Apache-2.0
+
+#include "ODCode93Patterns.h"
+
+namespace ZXing::OneD::Code93 {
+
+constexpr std::array<std::array<int, 6>, 48> CODE_PATTERNS = { {
```

Comment:
Here I meant move the definition of the variable into the header and drop the cpp-file completely. I'm not sure that adding `constexpr` at this location changes anything at all.